### ISSUE TITLE
kata-netmon does not work

### ISSUE THREAD

**Issue by yangfeiyu20102011** [kata-containers/kata-containers#2737]
**kata-runtime kata-network** is not used in Kata 2.x, but the code of kata-network sub command is kept in src/runtime/netmon/netmon.go.

```
kataCmd              = "kata-network"
kataCLIAddIfaceCmd   = "add-iface"
kataCLIDelIfaceCmd   = "del-iface"
...
func (n *netmon) handleRTMNewLink(ev netlink.LinkUpdate) error {
	...

	// Add the interface through the Kata CLI.
	if err := n.addInterfaceCLI(iface); err != nil {
		return err
	}

	...

	// Complete by updating the routes.
	return n.updateRoutes()
}
```

handleRTMNewLink will call kata-runtime kata-network, it does not work.

Why do we also keep it ?

**Comment by Jakob-Naucke** [comment]
> **kata-runtime kata-network** is not used in Kata 2.x, but the code of kata-network sub command is kept in src/runtime/netmon/netmon.go.

@yangfeiyu20102011 you may have tinkered more than I have, but enabling netmon in the config still seems to activate netmon. Are you sure?

**Comment by yangfeiyu20102011** [comment]
> > **kata-runtime kata-network** is not used in Kata 2.x, but the code of kata-network sub command is kept in src/runtime/netmon/netmon.go.
> 
> @yangfeiyu20102011 you may have tinkered more than I have, but enabling netmon in the config still seems to activate netmon. Are you sure?

Yeah, enabling netmon will activate netmon, but kata-netmon can't call kata-runtime kata-netowrk sub command to add the network interface.

**Comment by liubin** [comment]
Hi, the netmon is used to support Docker in Kata Containers 1.x, we are planning to remove it later.
